pages/linux/eww.md

```diff
@@ -0,0 +1,28 @@
+# eww
+
+> Implement your own custom widgets in any window manager.
+> More information: <https://elkowar.github.io/eww>.
+
+- Start the daemon:
+
+`eww daemon`
+
+- Open a widget:
+
+`eww -c {{path/to/source_code_directory}} open {{window_name}}`
+
+- Close a widget:
+
+`eww -c {{path/to/source_code_directory}} close {{window_name}}`
```

Comment:
From `eww --help`:
> -c, --config <CONFIG\> override path to configuration directory (directory that contains eww.yuck and eww.(s)css)
```suggestion
`eww -c {{path/to/configuration_directory}} open {{window_name}}`

- Close a widget:

`eww -c {{path/to/configuration_directory}} close {{window_name}}`
```

Reply:
For a new user I think it would be better to say "source_code".

The "configuration" _is_ technically the source code. Although the docs say this, I think a new user would understand "source code" better.

It's like having a js and css file, but instead of js it's a custom language (yuck).

You're not configuring it in the sense that there is a default widget that you edit or a configuration file, you build it from scratch (through the custom language). This is why I think its better to call it "source code".

"Yuck" isn't like yaml or toml or a .conf file it's a programming language.

Thoughts?

Reply:
OK, I've never used `eww`. Thanks for explaining.